hypervisor/arch/x86/guest/vcpuid.c

```diff
@@ -850,11 +855,11 @@ void guest_cpuid(struct acrn_vcpu *vcpu, uint32_t *eax, uint32_t *ebx, uint32_t
 
 		default:
 			/*
-			 * In this switch statement, leaf shall either be 0x01U or 0x0bU
-			 * or 0x0dU or 0x80000001U. All the other cases have been handled properly
-			 * before this switch statement.
-			 * Gracefully return if prior case clauses have not been met.
+			 * In this switch statement, leaf 0x01/0x0b/0x0d/0x19/0x80000001
+			 * shall be handled specifically. All the other cases
+			 * just return physical value.
 			 */
+			guest_cpuid_from_physical(leaf, eax, ebx, ecx, edx);
```

Comment:
This kind of emulation method is not the 1st time.  If we need to wrap in an API, we should do for all of them. But Given that is one line API. We can stay with direct call.